src/actions/MsgInfoActions.js

```diff
@@ -1,25 +1,25 @@
 import { MSG_INFO } from '../actionTypes/MsgInfoConstants';
 
 export const msgInfoActions = {
-  success: info => ({
+  success: message => ({
     type: MSG_INFO,
     payload: {
       success: true,
-      info
+      message
```

Comment:
change `info`  to `message` because the store itself is name info, so i think `info.message` is better than `info.info`